### PR TITLE
fix: '3-networks' minor inconsistencies and typos

### LINKED ISSUE #294: Minor '3-networks' inconsistencies
Few inconsistencies:

- In [3-networks/envs/development/main.tf](https://github.com/terraform-google-modules/terraform-example-foundation/blob/d6007d456ceb1b2ddf7e21a842ef5083bbe5586c/3-networks/envs/development/main.tf), projects data source is called [`base_project`](https://github.com/terraform-google-modules/terraform-example-foundation/blob/d6007d456ceb1b2ddf7e21a842ef5083bbe5586c/3-networks/envs/development/main.tf#L43), while other in environments it's called `base_host_project`([non-production](https://github.com/terraform-google-modules/terraform-example-foundation/blob/d6007d456ceb1b2ddf7e21a842ef5083bbe5586c/3-networks/envs/non-production/main.tf#L43), [production](https://github.com/terraform-google-modules/terraform-example-foundation/blob/d6007d456ceb1b2ddf7e21a842ef5083bbe5586c/3-networks/envs/production/main.tf#L43)). Should we make it `base_host_project` everywhere?
- In [3-networks/envs/development/main.tf](https://github.com/terraform-google-modules/terraform-example-foundation/blob/d6007d456ceb1b2ddf7e21a842ef5083bbe5586c/3-networks/envs/development/main.tf#L93-L103) and [3-networks/envs/production/main.tf](https://github.com/terraform-google-modules/terraform-example-foundation/blob/d6007d456ceb1b2ddf7e21a842ef5083bbe5586c/3-networks/envs/production/main.tf#L93-L103), when creating restricted shared VPC, secondary ranges use `default_region1`, while [3-networks/envs/non-production/main.tf](https://github.com/terraform-google-modules/terraform-example-foundation/blob/d6007d456ceb1b2ddf7e21a842ef5083bbe5586c/3-networks/envs/non-production/main.tf#L94-L104) use `default_region2`. Shall we use `default_region1` everywhere?
- In [3-networks/envs/development/main.tf](https://github.com/terraform-google-modules/terraform-example-foundation/blob/d6007d456ceb1b2ddf7e21a842ef5083bbe5586c/3-networks/envs/development/main.tf#108) comment states `Base shared VPC`, while for `non-production` and `production` it's `Private shared VPC`. Module name is `base_shared_vpc`. Shall we make comment consistent with the module name? Also, in `secondary_ranges` definition there, code style differs a bit.
- In all `variables.tf`, there's a typo [`Subnetwoks`](https://github.com/terraform-google-modules/terraform-example-foundation/blob/d6007d456ceb1b2ddf7e21a842ef5083bbe5586c/3-networks/envs/shared/variables.tf#L45).
- In `variables.tf` for `development`/`non-production`/`production` environments - there's a typo [`logginglogging`](https://github.com/terraform-google-modules/terraform-example-foundation/blob/d6007d456ceb1b2ddf7e21a842ef5083bbe5586c/3-networks/envs/production/variables.tf#L61).

If that all looks legit, I can create a PR to fix it. Thanks!

### PR DESCRIPTION
**Pull request by vovinacci**
This PR fixes `3-networks` minor inconsistencies, typos and copy-pastas.

Closes #294

### CODE REVIEW

**Comment by vovinacci** [comment]
@rjerrems @bharathkkb Integration test failed, checked the test spec - didn't find anything that may have broken by this PR. As I don't have access to cloud build for this project, could you please take a look on what it failed? Much appreciated.

**Comment by rjerrems** [comment]
I had a look and triggered the test again, it looks like we hit a transient issue on both. First was a concurrent operation updating IAM on billing project and second was a clash in terms of project names.

@bharathkkb - Did we modify the prefix already? Is that only on develop?

For now I have triggered another test run

**Comment by vovinacci** [comment]
@rjerrems Thanks much! Seems to be fine now.

File: 3-networks/envs/development/main.tf [hunk 1]
```diff
@@ -105,7 +105,7 @@ module "restricted_shared_vpc" {
 }
 
 /******************************************
- Base shared VPC
```
Comment (rjerrems) [inline 3-networks/envs/development/main.tf:108]: This should stay as Base as per page 30 of the security foundation guide here https://services.google.com/fh/files/misc/google-cloud-security-foundations-guide.pdf

**Comment by rjerrems** [comment]
Thanks @vovinacci - I have one small nit for a comment, but once that's fixed it LGTM

**Comment by vovinacci** [comment]
@rjerrems Thanks a lot for the review. Reverted back to `Base`, also amended for other environments, so the naming is consistent.

**Review by rjerrems — APPROVED** [review]
LGTM

@bharathkkb FYI

**Pull request merged**